File: 2714-left-and-right-sum-differences/left-and-right-sum-differences.py
class Solution:
    def leftRightDifference(self, nums: List[int]) -> List[int]:
        output = []
        for i in range(0,len(nums)):
            L = sum(nums[0:i])         
            R = sum(nums[i+1:])
            output.append(abs(L- R))
        return output


        

File: 2714-left-and-right-sum-differences/left-and-right-sum-differences.cs
public class Solution {
    public int[] LeftRightDifference(int[] nums) {
        int lsum = 0, rsum = 0, output = 0;
        for(int i = 0; i < nums.Length;i++){
            rsum += nums[i];
        }
        for(int i = 0; i < nums.Length;i++){
            rsum -= nums[i];
            output = Math.Abs(lsum - rsum);
            lsum += nums[i];
            nums[i] = output;
        }
        return nums;

    }
}